infrastructure_deployment_files/1_kube_via_NodePort/CLI_Commands_Docker_Kube/CLI_commands_app_todo_DOCKER

```diff
@@ -0,0 +1,31 @@
+BUILD
+docker build -t todo-app:prod .
```

Comment:
consider convert to markdown file `.md`. For reference see `README.md`

example


## Build
```shell
docker build -t todo-app:prod .
docker run -d -p 8080:8080 todo-app:prod
```